scripts/calibnet_health_check.sh

```diff
@@ -30,12 +30,22 @@ echo "$1" > preloaded_wallet.key
 
 echo "Fetching params"
 $FOREST_CLI_PATH fetch-params --keys
+echo "Downloading zstd compressed snapshot without aria2"
+$FOREST_CLI_PATH --chain calibnet snapshot fetch --provider filecoin --compressed -s "$SNAPSHOT_DIRECTORY"
+echo "Downloading snapshot without aria2"
+$FOREST_CLI_PATH --chain calibnet snapshot fetch --provider filecoin -s "$SNAPSHOT_DIRECTORY"
+echo "Cleaning up snapshots"
+$FOREST_CLI_PATH --chain calibnet snapshot clean -s "$SNAPSHOT_DIRECTORY" --force
+echo "Cleaning up snapshots again"
+$FOREST_CLI_PATH --chain calibnet snapshot clean -s "$SNAPSHOT_DIRECTORY" --force
```

Comment:
Why do we need to repeat `snapshot clean` again?

Reply:
It's a good question. It might be worth adding this issue in the code to make it clear from the start.